src/files/BrsFile.spec.ts

```diff
@@ -2463,6 +2463,30 @@ describe('BrsFile', () => {
     });
 });
 
+export function trimSource(source: string, fixIndent = true) {
```

Comment:
Can this be removed in favor of the `trim` function mentioned earlier?

Reply:
This method doesn't do the same as trim I think - it does normalize indentation.

Reply:
Oh gotcha it does normalise indentation too.